sdk/dapp-kit/src/components/styling/StyleMarker.css.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { globalStyle } from '@vanilla-extract/css';
+
+import { styleDataAttributeSelector } from '../../constants/styleDataAttribute.js';
+
+globalStyle(createScopedSelector('*'), {
+	boxSizing: 'border-box',
+});
+
+globalStyle(createScopedSelector('button'), {
+	appearance: 'none',
```

Comment:
I found it better to manually reset the styles we need to vs using `all: unset` which was setting some stuff to `initial` that I wanted to be `inherit`